Artroom1/Controller/SavedArtViewController.swift

```diff
@@ -12,12 +12,12 @@ import UIKit
 class SavedArtViewController: UIViewController {
     var arrayOfSavedArt: [Artwork]?
     var savedArtImage: UIImageView!
-
     // Displays carousel of artworks - for now, images only
     @IBOutlet var savedArt: iCarousel?
-
     override func viewDidLoad() {
         super.viewDidLoad()
+        arrayOfSavedArt = arrayOfSavedArt!.uniqueElements
```

Comment:
Can you explain me the reasoning here, why do we need `.uniqueElements`?

Reply:
This one is because a user can technically go back and save the same piece of art over and over. With this, we make sure it only shows up in the Saved Art room once.

Reply:
I see. In that case I would suggest that you prevent this problem at the source and not later on. So the original dataset of user likes should not contain duplicates.